pkg/login/social/github_oauth.go

```diff
@@ -47,6 +47,10 @@ func (s *SocialGithub) IsSignupAllowed() bool {
 	return s.allowSignup
 }
 
+func (s *SocialGithub) OrgToRoleMap() map[string][]SocialGroup {
+  return nil
+}
+
```

Comment:
I think we should change other oauth providers if not needed in order to keep the patch as small as possible and make it easier to merge upstream changes

Reply:
Each provider implements a SocialConnector interface.  The function needs to be added in each, even if not used.